modules/users/client/components/UserDoesNotExist.component.js

```diff
@@ -9,7 +9,7 @@ export default function UserDoesNotExist() {
     <div className="row text-center" role="alert">
       <h1>{t('Oops!')}</h1>
       <em className="lead">
-        {t('Member you are looking for is not available.')}
+        {t('The Trustrooter you are looking for is not available.')}
```

Comment:
@chagai95 thanks for suggestion! What do you think if we'd use "Trustroots member" instead? It would be a dozen times easier to translate to different languages and little less confusing for those who are bad in English.

Reply:
Sounds great! It still needs the "the" though otherwise it sounds a bit weird in English... The Trustroot's member or maybe even just a simple "the person"

Reply:
👍 Sure! I'll update and merge.